sklearn/utils/_param_validation.py

```diff
@@ -168,6 +169,8 @@ def decorator(func):
 
         @functools.wraps(func)
         def wrapper(*args, **kwargs):
+            if get_config()["skip_parameter_validation"]:
+                return func(*args, **kwargs)
```

Comment:
I wonder my codecov does not complain that this line is not covered by the test without an extra smoke test.

Reply:
Hum, this is probably because of the `_minibatch_step` call that has been context managed in this PR.

Reply:
yes :)